manager/integration/tests/test_basic.py

```diff
@@ -467,6 +467,116 @@ def snapshot_test(clients, volume_name, base_image):  # NOQA
 
     cleanup_volume(client, volume)
 
+@pytest.mark.coretest   # NOQA
+def test_backup_status_for_unavailable_replicas(clients, volume_name):    # NOQA
+    """
+    Test backup status for unavailable replicas
+
+    Setup:
+
+    1. Create a volume and attach to the current node
+    2. Run the test for all the available backupstores.
+
+    Steps:
+
+    1. Create a backup of volume
+    2. Find the replica for that backup
+    3. Disable scheduling on the node of that replica
+    4. Delete the replica
+    5. Wait for volume backup status state to go to error
+    6. Verify backup status error contains `unknown replica`
```

Comment:
Explain a little more on why this will result in `unknown replicas`, e.g. why we need to disable the scheduling etc.

Reply:
Doesn't really fit under the Steps heading, should I add an explanation/context Heading?

Reply:
Yeah, that's better